targets/HUDL/write_to_display.cpp

```diff
@@ -0,0 +1,71 @@
+//
+// Created by aaron on 9/1/22.
+//
+
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/manager.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/utils/time.hpp>
+#include <HUDL/HUDL.hpp>
+
+namespace IO = EVT::core::IO;
+namespace time = EVT::core::time;
+
+constexpr uint32_t SPI_SPEED = SPI_SPEED_4MHZ; // 4MHz
+constexpr uint8_t deviceCount = 1;
+
+int main() {
+  IO::GPIO *devices[deviceCount];
```

Comment:
This should propbably references instead of pointers.

Reply:
Can you explain more? Why should this be a reference if we are initializing a buffer to store a device.

Reply:
Pointers are more dangerous than references. If you can instantiate as a reference instead of a pointer, you should. [(https://stackoverflow.com/questions/7058339/when-to-use-references-vs-pointers)]